citizen/citizen.c

```diff
@@ -438,52 +485,9 @@ Citizen getOldestCitizen(List *list) {
  * @param list A pointer to the list of citizens.
  * @return The youngest citizen in the list.
  */
-Citizen getYoungestCitizen(List *list) {
-    // Sort the list by birth date in ascending order
-    List sortedBdate = sortCitizen(list, BirthDate);
-    Citizen youngestCitizen = sortedBdate.tail->citizen; // Last citizen in sorted list is the youngest
-    return youngestCitizen;
-}
-
-/**
- * @brief View demographic information of the citizen list.
- *
- * This function prints the total population, male population, female population, the oldest citizen, and the youngest citizen.
- *
- * @param list A pointer to the list of citizens.
- */
-void viewDemographics(List *list) {
-    // Check if the list is empty
-    if (list == NULL || list->head == NULL) {
-        printf("No demographic information available. The list is empty.\n");
-        return;
-    }
-
-    // Get demographic details
-    int totalPopulation = counter; // Assuming 'counter' holds total number of citizens
-    int totalMalePopulation = getMalePopulation(list);
-    int totalFemalePopulation = getFemalePopulation(list);
-
-    // Get oldest and youngest citizens
-    Citizen oldestCitizen = getOldestCitizen(list);
-    Citizen youngestCitizen = getYoungestCitizen(list);
-
-    // Display the demographic information
-    printf("Demographic Information\n");
-    printf("Total Population: %d\n", totalPopulation);
-    printf("Total Male Population: %d\n", totalMalePopulation);
-    printf("Total Female Population: %d\n", totalFemalePopulation);
-
-    // Print details of the oldest citizen
-    char oldestName[200];
-    snprintf(oldestName, sizeof(oldestName), "%s %s", oldestCitizen.firstName, oldestCitizen.lastName);
-    printf("The Oldest Citizen: %s\n", oldestName);
-    printf("Age: %d\n", getCitizenAge(list, oldestCitizen.citizenId));
-
-    // Print details of the youngest citizen
-    char youngestName[200];
-    snprintf(youngestName, sizeof(youngestName), "%s %s", youngestCitizen.firstName, youngestCitizen.lastName);
-    printf("The Youngest Citizen: %s\n", youngestName);
-    printf("Age: %d\n", getCitizenAge(list, youngestCitizen.citizenId));
-}
-
+Citizen getyoungest_citizen(List *list)
```

Comment:
please me consistent, the previous uses camelCase on the funtion naming convention

Reply:
Applied, thanks.